### ISSUE TITLE
Inception V3 Architecture Diagram is not useful

### ISSUE THREAD

**Issue by sudhashbahu** [tensorflow/models#545]
## Please let us know which model this issue is about (specify the top-level directory)

We have the Architecture diagram at https://github.com/tensorflow/models/blob/master/inception/g3doc/inception_v3_architecture.png

However there are no operation/tensor names so the information is not very helpful if we want to trace back through the model.

Please update the model so that it is clear that the output tensor is softmax:0 and the one before that is pool_3:0 and so on.

**Comment by metacomgd** [comment]
See here:
http://stackoverflow.com/questions/35336648/list-of-tensor-names-in-graph-in-tensorflow/

**Comment by sudhashbahu** [comment]
That link does not answer the questions.

Instead of everyone having to write programs to get the info, why not publish the information along with the model and save everyone the effort?

**Comment by asimshankar** [comment]
@sudhashbahu :The architecture in the README is intended to be a very high level overview. Adding operation names for each of the nodes will likely be overwhelming. Note that you can visualize any graph in [TensorBoard](https://www.tensorflow.org/how_tos/graph_viz/index.html#tensorboard-graph-visualization)

Unfortunately, yes, this does mean that to visualize you either need to write some code, or execute TensorBoard and point it to training summaries. At least that gives the most accurate visualization and names of operations.

Since we don't plan to add a whole lot to the architecture diagram, I'll close this issue out for now. Do let me know if you have concerns.

FYI, the following code dumps out the graph in python in a form suitable for visualizing in tensorboard (and the hosted [demo instance](https://www.tensorflow.org/tensorboard/index.html))
```python
with open('/tmp/graph.pb.txt', 'w') as f:
  f.write(str(tf.get_default_graph().as_graph_def())
```
